src/commands/dev-env-import-sql.js

```diff
@@ -25,9 +28,9 @@ export class DevEnvImportSQLCommand {
 		this.slug = slug;
 	}
 
-	async run( silent = false ) {
+	async run() {
```

Comment:
This is gonna be an issue: https://github.com/Automattic/vip-cli/blob/054cd85b5fc9a008f300342787437e7ab1bdd432/src/commands/dev-env-sync-sql.js#L177-L182

Also, I'd prefer keeping the `silent` parameter because it can be useful in re-purposing a command class in multiple places. I think all the command-classes' `run` functions have this, for this purpose.

Reply:
I have updated `dev-env-sync-sql.js` to pass `quiet: true` in the constructor's parameters.

This is because, with VIP CLI Express, we won't be able to pass arbitrary arguments to `run()`. The correct approach will be to instantiate the command with the necessary parameters.